### ISSUE TITLE
Network analysis sub_page

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#381]
Have a sub_page for network analyze
Nodes
- number of nodes in the network
- number of triangles in the network
- max node connection
- number of one connection or end node
Edges
- number of edges
- max edge weight
- min edge weight
- avg edge weight

I used NetworkX for doing my analysis. there are others
https://www.timlrx.com/2019/05/05/benchmark-of-popular-graph-network-packages/

**Comment by reza1615** [comment]
Here is the notebook for getting above metrics

[NetworkX.zip](https://github.com/man-group/dtale/files/5758284/NetworkX.zip)

**Comment by reza1615** [comment]
+ Shortest path between node 'z' and 'j'

`nx.shortest_path(G, source='z', target='j')`

**Comment by aschonfeld** [comment]
@reza1615 I think I've finalized all the functionality you have asked for.  Here it is:

https://user-images.githubusercontent.com/11547371/103466035-40af2380-4d0f-11eb-9ae4-fa5535507bcf.mov

FYI: to do "Shortest Path" you would Shift+Click on two nodes and the path will display at the top of the screen.  I've also added instructions on how the page works you would just expand the section at the top marked "Network Viewer"

If there is nothing glaring I'll work on getting some coverage on this code and then release it to the masses.
